packages/core/android/src/main/java/com/datadog/reactnative/DdRum.kt

```diff
@@ -82,7 +113,14 @@ class DdRum(reactContext: ReactApplicationContext) : ReactContextBaseJavaModule(
      */
     @ReactMethod
     fun addAction(type: String, name: String, context: ReadableMap, timestampMs: Double, promise: Promise) {
-        nativeInstance.addAction(type, name, context.toHashMap(), timestampMs.toLong())
+        val attributes = context.toHashMap().toMutableMap().apply {
+            put(RumAttributes.INTERNAL_TIMESTAMP, timestampMs.toLong())
+        }
+        GlobalRum.get().addUserAction(
+            type = type.asRumActionType(),
```

Comment:
shouldn't the `type` be an enum here ? Is this available in TS ?

Reply:
Yes, but I'm not sure we can keep enum consistency between ts and native for now. 
I'll dig to see what's doable right now, and I think it's something that is going to be achievable with the new React Native architecture.